components/CommonLinks/CommonLinks.module.css

```diff
@@ -1,7 +1,49 @@
 .CommonLinks__container {
-    padding: 15px;
+    position: fixed;
+    display: flex;
+    gap: 12px;
```

Comment:
Let's use either margin for children, or gap for the parent

Reply:
Indents were not needed at all here 🙂 

P.S I will remember about the choice between gap and margin